packages/fxa-auth-server/lib/senders/emails/templates/passwordForgotOtp/index.mjml

```diff
@@ -0,0 +1,37 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="password-forgot-otp-title">Your confirmation code</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="password-forgot-otp-request">
+        We received a request to reset your password. Enter this code to confirm it’s really you.
+      </span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+
+    <mj-text css-class="code-large"><%- code %></mj-text>
+
+    <mj-text css-class="text-body-no-margin">
+      <span data-l10n-id="password-forgot-otp-expiry-notice">This code expires in 5 minutes.</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-body-subtext">
+      <%- include('/partials/requestDevice/index.mjml') %>
+      <%- include('/partials/automatedEmailNoAction/index.mjml') %>
```

Comment:
This should probably need to be `automatedEmailResetPassword` since the user will need to take action if they didn't initate the request.

Reply:
@vbudhram currently `automatedEmailResetPassword` is used after the password's been reset so it's not quite the same?

Reply:
`automatedEmailNoAction` sounds right to me in this context, as in the `recovery` email template 👍 

![image](https://github.com/mozilla/fxa/assets/22231637/edc29f1f-6d2c-481c-8b4e-47f86b940613)